lib/middleware/request-handler.ts

```diff
@@ -406,11 +412,17 @@ export function elevateRequest (req: Interface.IOurRequest,
         res.write(text + JSON.stringify(data));
     };
 
-    res.sendEnd = (status: string, message: string): void => {
-        var text = util.format('%s%s,%s}',
+    res.sendEnd = (status: string,
```

Comment:
As a client of using WebSockets/SocketIO, I would advocate for only providing the array of correlation ids.